### PR TITLE
Add script load organization data

### PR DESCRIPTION
**Pull request by azimhamza**
## What this PR accomplishes

This pull request updates the `README.md` file to provide clearer instructions and adds the `load_organization_data.py` script to the repository in scripts. The changes aim to improve the usability and functionality of the project.

## Issue(s) addressed

This PR addresses the following issues:

- [GitHub Issue #123](https://github.com/ongov/ckanext-ontario_theme/commit/6373a9ba25359b95d3c6f8ccb07ce0c3a351c6ae) - Updating README and adding load_organization_data.py script

## What needs review

Please review the changes made in this pull request and provide feedback on the following:

1. README.md Updates:
   - Verify that the updated instructions and information in the README.md file are clear and easy to follow.
   - Ensure that the prerequisites, setup instructions, and usage details are accurate and complete.
   - Check for any typos, grammar mistakes, or formatting issues in the README.md file.

2. load_organization_data.py Script:
   - Review the implementation of the `load_organization_data.py` script and its adherence to coding standards.
   - Check if the script handles various scenarios and edge cases appropriately.
   - Assess the script's efficiency and performance, especially when processing large CSV files.
   - Ensure that the script accurately creates organizations and uploads associated resources to the CKAN application.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+3. Locate the `api_token` variable in the script and replace the empty string `""` with your obtained API key:
+```python
+api_token = "your_api_key_here"
+
+
+## Usage 
+
+Ensure that your CKAN application is running locally at http://localhost.
+
+Place the CSV files containing organization data in the uploads directory within the repository.
```
Comment (aimalkhan) [inline scripts/README.md:108]: is this needed? the script's method `get_some_packages_by_format` downloads the org data to create the CSV and then pushes it to local

File: scripts/README.md [hunk 2]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+2. Open the `load_organization_data.py` script in a text editor.
+
+3. Locate the `api_token` variable in the script and replace the empty string `""` with your obtained API key:
+```python
+api_token = "your_api_key_here"
+
+
+## Usage 
+
+Ensure that your CKAN application is running locally at http://localhost.
```
Comment (aimalkhan) [inline scripts/README.md:106]: http://localhost  -> `http://localhost`

File: scripts/README.md [hunk 3]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+
+Place the CSV files containing organization data in the uploads directory within the repository.
+
+Open a terminal or command prompt and navigate to the repository directory.
+
+Run the script using the following command:
+
+python scripts/load_organization_data.py
```
Comment (aimalkhan) [inline scripts/README.md:114]: python scripts/load_organization_data.py -> `$ python scripts/load_organization_data.py`

File: scripts/README.md [hunk 4]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+1. Obtain an API key from your CKAN application. You can find the API key in the CKAN user settings at `http://localhost/user/edit/admin`. Make sure you are logged in as an administrator.
+
+2. Open the `load_organization_data.py` script in a text editor.
+
+3. Locate the `api_token` variable in the script and replace the empty string `""` with your obtained API key:
+```python
+api_token = "your_api_key_here"
+
+
+## Usage
```
Comment (aimalkhan) [inline scripts/README.md:104]: the content in this section is all bullet-points/steps to follow. Make sure they are bullet points. Use the other section in the document as a reference for styling.

File: scripts/README.md [hunk 5]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+
+
+The script will process the CSV files and create organizations in your CKAN application. Any resources associated with the organizations will be uploaded and attached to the respective organizations.
+
+Monitor the script's output and check for any error messages or failed resource uploads.
+
+Once the script completes, verify that the organizations and their associated resources have been successfully created in your CKAN application.
+
+## Additional Notes
+The script expects the CSV files to be present in the uploads directory. Ensure that the CSV files follow the required format and contain the necessary data for creating organizations and resources.
```
Comment (aimalkhan) [inline scripts/README.md:124]: CSV is not needed. The script's method get_some_packages_by_format downloads the org data to create the CSV and then pushes it to local

File: scripts/README.md [hunk 6]
```diff
@@ -70,3 +70,61 @@ Check that Solr is running on `http://127.0.0.1:8983/solr/`. The ckan core shoul
...
+
+If any errors occur during resource uploads, review the error messages and the failed_resource_writes list in the script's output. You may need to investigate and address any issues with the resource files or network connectivity.
+
+You can customize the script further to accommodate specific requirements or modify the behavior as needed.
```
Comment (aimalkhan) [inline scripts/README.md:128]: not needed

**Review by aimalkhan — CHANGES_REQUESTED** [review]
need a few styling and factual fixes before merge

**Comment by aimalkhan** [comment]
Can you also change the PR's title to be readable and in line with other PRs in the repo?

**Comment by aimalkhan** [comment]
> Can you also change the PR's title to be readable and in line with other PRs in the repo?

`Data-921:add_script_load_organization_data` should be `Add script to load test data into local env`, which will be inline with the rest of the PRs

File: scripts/README.md [hunk 7]
```diff
@@ -1,72 +1,118 @@
...
+### Prerequisites
+- Python 3.x
+- Requests library (install via `pip install requests`)
+
+### Script Setup
+1. Clone the repository containing the script.
+2. Navigate to the repository directory and install the required dependencies by running:
 ```
-$ curl -s -o /dev/null -I -w '%{http_code}' http://{ip_address}:8983/solr/admin/cores?action=STATUS
+pip
```
Comment (aimalkhan) [inline scripts/README.md:85]: typo in this script command
image

**Review by aimalkhan — CHANGES_REQUESTED** [review]
1 last change needed before merging.

File: scripts/README.md (hunk 7 above)
Comment (azimhamza) [reply scripts/README.md:85]: my bad

**Review by aimalkhan — APPROVED** [review]

**Pull request merged**